site/frontend/src/pages/status_new/page.vue

```diff
@@ -97,6 +98,21 @@ function formatStatus(status: BenchmarkRequestStatus): string {
   }
 }
 
+function hasErrors(errors: Dict<string>) {
+  // This is a bit odd but if the error object has values the loop will be
```

Comment:
You already use `Object.keys(errors).length` in `getErrorsLength`, you can use the same here :)

Reply:
6913885c263c5febfb6cdb241c7b5a8098a4b681 :+1: